client/components/admin_form/index.coffee

```diff
@@ -30,3 +30,7 @@ module.exports = class AdminEditView extends Backbone.View
 
   ignoreUnsaved: ->
     window.onbeforeunload = ->
+
+module.exports.init = ->
+  new AdminEditView
+    el: $('body')
```

Comment:
@kanaabe - I was experiencing the same problem of an empty el on the AdminEditView -- seems to be working on the other curation, which is the only place I saw the class used.